pkg/subctl/cmd/validate_deployment.go

```diff
@@ -20,13 +20,14 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 var validatePodsCmd = &cobra.Command{
-	Use:   "pods",
+	Use:   "pod-status",
```

Comment:
I prefer `pods` - shorter.

```suggestion
	Use:   "pods",
```

Reply:
I think `pod-status` is better describe the action.
@sridhargaddam  - what do you think?

Reply:
I don't think the `status` part is necessary plus it's singular (which pod?).  I think `subctl validate pods`  is shorter, clean and properly conveys the action of verifying the existence and state of the pods.

Reply:
I'm about to modify the same file to include checks for Overlapping CIDRs. I discussed with @aswinsuryan and we felt that we can rename the command from `pods` to `deployment`. 

So basically, as part of `subctl validate deployment` we will be checking if the desired number of Submariner pods are running, the state of the Pods and if there are any issues related to Overlapping CIDRs. So, IMHO a broader term like `deployment` might be apt. If you have any suggestions based on this updated info, please comment.

Reply:
`deployment` crossed my mind too since we check the `Deployments` and `DaemonSets` too. Sounds good to me.

Reply:
The following PR validates the CIDRs - https://github.com/submariner-io/submariner-operator/pull/1176